spec.html

```diff
@@ -26008,7 +25473,6 @@ <h1>Object.values ( _O_ )</h1>
       <h1>Properties of the Object Prototype Object</h1>
       <p>The Object prototype object:</p>
       <ul>
```

Comment:
I think we should keep:
```suggestion
      <ul>
        <li>is the intrinsic object <dfn>%Object.prototype%</dfn>.</li>
```

Reply:
When I posted <https://github.com/tc39/ecma262/pull/2056#pullrequestreview-436929886>, I thought that you’d add the `<dfn>`s for the remaining `%*.prototype%` properties without me needing to list them manually.